<ui version="4.0" >
 <class>UIProjectsManager</class>
 <widget class="QDockWidget" name="UIProjectsManager" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Projects Manager</string>
  </property>
  <widget class="QWidget" name="wCentral" >
   <layout class="QVBoxLayout" >
    <property name="spacing" >
     <number>3</number>
    </property>
    <property name="leftMargin" >
     <number>5</number>
    </property>
    <property name="topMargin" >
     <number>5</number>
    </property>
    <property name="rightMargin" >
     <number>5</number>
    </property>
    <property name="bottomMargin" >
     <number>5</number>
    </property>
    <item>
     <widget class="QFrame" name="fButtons" >
      <property name="minimumSize" >
       <size>
        <width>16</width>
        <height>25</height>
       </size>
      </property>
      <property name="frameShape" >
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>23</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="tbFiltered" />
       </item>
       <item>
        <widget class="QToolButton" name="tbNew" />
       </item>
       <item>
        <widget class="QToolButton" name="tbOpen" >
         <property name="popupMode" >
          <enum>QToolButton::MenuButtonPopup</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="l1" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbSave" />
       </item>
       <item>
        <widget class="QToolButton" name="tbSaveAll" />
       </item>
       <item>
        <widget class="Line" name="l2" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbClose" />
       </item>
       <item>
        <widget class="QToolButton" name="tbCloseAll" />
       </item>
       <item>
        <widget class="Line" name="l3" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbSettings" />
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="sProjects" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget" >
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>3</number>
        </property>
        <item>
         <widget class="QLabel" name="lOpenedProjects" >
          <property name="text" >
           <string>Opened Projects :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="tvOpenedProjects" />
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget" >
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>3</number>
        </property>
        <item>
         <widget class="QLabel" name="lProjects" >
          <property name="text" >
           <string>Current Project(s) :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="tvProjects" />
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../resources/resources.qrc" />
 </resources>
 <connections/>
</ui>
